### PR TITLE
Add br.com.eduk.app

### PR DESCRIPTION
**Pull request by greghorymetzker**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

**Review by hfiguiere — CHANGES_REQUESTED** [review]
Please use English as the default language for the metainfo and .desktop file. You can add the translation as needed.

File: br.com.eduk.app.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  <project_license>GPL-3.0-only</project_license>
+  
+  <description>
+    <p>
+      Tenha acesso a um catálogo completo de cursos e oportunidades para gerar renda!
+    </p>
+  </description>
+  
+  <launchable type="desktop-id">br.com.eduk.app.desktop</launchable>
+</component>
```
Comment (hfiguiere) [inline br.com.eduk.app.metainfo.xml:18]: missing `release`, `developer_name`, `content_rating`, `url`.

File: br.com.eduk.app.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+app-id: br.com.eduk.app
```
Comment (hfiguiere) [inline br.com.eduk.app.yml:1]: `app` is NOT an appropriate name

File: flathub.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+    "only-arches": [
+        "x86_64"
```
Comment (hfiguiere) [inline flathub.json:3]: why no `aarch64` ?

File: br.com.eduk.app.yml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+base: org.electronjs.Electron2.BaseApp
+base-version: '23.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
```
Comment (hfiguiere) [inline br.com.eduk.app.yml:14]: you probably need `--device=dri` since it's Electron based

File: br.com.eduk.app.metainfo.xml (hunk 1 above)
Comment (greghorymetzker) [reply br.com.eduk.app.metainfo.xml]: Fix

File: br.com.eduk.app.yml (hunk 2 above)
Comment (greghorymetzker) [reply br.com.eduk.app.yml]: Fix

File: flathub.json (hunk 3 above)
Comment (greghorymetzker) [reply flathub.json:3]: The company currently does not have this architecture.

File: br.com.eduk.app.yml (hunk 4 above)
Comment (greghorymetzker) [reply br.com.eduk.app.yml]: Fix

File: .DS_Store
Comment (bbhtt) [inline .DS_Store:1]: Delete

File: br.com.eduk.desktop
Comment (bbhtt) [inline br.com.eduk.desktop:1]: Desktop file should be upstream

File: br.com.eduk.metainfo.xml [hunk 5]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>br.com.eduk</id>
+  
+  <name>eduK</name>
+  <summary>A revolution in the social impact technology market, combining online education and income generation.</summary>
```
Comment (bbhtt) [inline br.com.eduk.metainfo.xml:6]: Remove the article "A" in front and dot in the end.

Also the line doesn't describe what the app is for/does...

File: br.com.eduk.metainfo.xml
Comment (bbhtt) [inline br.com.eduk.metainfo.xml:1]: This file should be submitted upstream

File: flathub.json (hunk 3 above)
Comment (bbhtt) [reply flathub.json:3]: You're building from source, if it builds and runs you can remove the restriction.

File: br.com.eduk.yml [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+      # Install app wrapper
+      - install -Dm755 -t /app/bin/ ../run.sh
+      - install -Dm444 logo.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm444 ./flatpak/${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm444 ./flatpak/${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+    subdir: main
+    sources:
+      # Getting project
+      - type: git
+        url: https://github.com/greghorygusmao/br.com.eduk.app.git
```
Comment (bbhtt) [inline br.com.eduk.yml:49]: The link is 404... https://github.com/greghorygusmao/br.com.eduk.app

File: br.com.eduk.metainfo.xml [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+  <url type="homepage">https://app.eduk.com.br</url>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+  
+  <description>
+    <p>
+      Founded in 2013 and acquired by Mind Lab in 2021, eduK aims to address Brazil&apos;s dual challenge of future-focused education and rectifying past deficits through income generation. With the acquisition, Mind Lab set a goal to enable a million users to generate income by 2024, offering affordable access to over 3,200 courses and 9,000 hours of content from 1,100 experts to enhance skills and knowledge on any device. EduK has impacted over 5 million people and was recognized by HolonIQ in 2020 as one of Latin America&apos;s most innovative EdTech startups.
+    </p>
```
Comment (hfiguiere) [inline br.com.eduk.metainfo.xml:15]: I don't understand what this application does.

File: br.com.eduk.yml [hunk 8]
```diff
@@ -0,0 +1,60 @@
+app-id: br.com.eduk
```
Comment (bbhtt) [inline br.com.eduk.yml:1]: How are you related with https://app.eduk.com.br/ ? I don't see any mention of this software on that webpage.

File: br.com.eduk.yml (hunk 6 above)
Comment (hfiguiere) [reply br.com.eduk.yml:49]: Probably mean the repository is private.

File: flathub.json (hunk 3 above)
Comment (greghorymetzker) [reply flathub.json:3]: Fix

File: br.com.eduk.desktop
Comment (greghorymetzker) [reply br.com.eduk.desktop:1]: Fix

File: br.com.eduk.metainfo.xml (hunk 5 above)
Comment (greghorymetzker) [reply br.com.eduk.metainfo.xml:6]: Fix

File: br.com.eduk.metainfo.xml
Comment (greghorymetzker) [reply br.com.eduk.metainfo.xml:1]: Fix

File: br.com.eduk.yml (hunk 6 above)
Comment (greghorymetzker) [reply br.com.eduk.yml:49]: Fix

File: br.com.eduk.metainfo.xml (hunk 7 above)
Comment (greghorymetzker) [reply br.com.eduk.metainfo.xml:15]: eduK is a freemium SaaS educational platform, built with Electron, focused on online vocational courses. It offers over 3,200 courses, adopting a model that combines free access to selected content with subscription options for advanced features. Aimed at professional training, this is evidenced by the issuance of more than 100,000 certificates, and it promotes the professional success of its users by facilitating employment opportunities and business partnerships.

File: br.com.eduk.metainfo.xml [hunk 9]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>br.com.eduk</id>
```
[the remark this answers is not in the document]
Comment (greghorymetzker) [reply br.com.eduk.metainfo.xml:3]: Fix

File: br.com.eduk.yml (hunk 8 above)
Comment (greghorymetzker) [reply br.com.eduk.yml:1]: I serve as the product leader in this company and have recently uploaded a new update of the project to the EduK company's repository. This action strengthens my connection with the organization and demonstrates ongoing commitment to the development and enhancement of our solutions.

**Pull request closed without merging**